searchlib/src/vespa/searchlib/tensor/hnsw_index_explorer.cpp

```diff
@@ -63,6 +71,30 @@ HnswIndexExplorer<type>::get_state(const vespalib::slime::Inserter& inserter, bo
                    cfg.neighbors_to_explore_at_construction());
 }
 
+template <HnswIndexType type>
+std::vector<std::string>
+HnswIndexExplorer<type>::get_children_names() const
+{
+    return { LEVELS_STORE_NAME, LINKS_STORE_NAME, NODEID_STORE_NAME };
```

Comment:
To be symmetric with `get_child`, should `NODEID_STORE_NAME` only be returned if `type == HnswIndexType::MULTI`?

Reply:
The name is effectively hidden if get_child returns an empty unique_ptr for the name. The same can bee seen in `Proton::get_children_names` vs `Proton::get_child`.